### DISCUSSION TITLE
Unifi Protect RTSPS feeds

### DISCUSSION THREAD

**Discussion by patdemko** [blakeblackshear/frigate#3260] [General]
Hi all,

If anyone sends RTSPS feeds from their Unifi Protect application to Frigate you probably do not want to upgrade to the latest UniFi Protect Application 2.0.0 version that just came out.  After I upgraded all of my RTSPS feeds into Frigate broke.  There's nothing in the release notes about RTSP issues with this update, but reading forum posts there's been a few people complaining on issues with RTSP during the beta as well as after the final release.  So far I haven't found a solution to this. Just figured I'd try to give a warning to anyone that may run into the same issue.

**Comment by NickM-27** [comment]
See this issue for more context as well as ways to fix it https://github.com/blakeblackshear/frigate/issues/3067

**Reply by patdemko** [reply]
Thanks for this!  I had already tried the "- an" flag and it didn't work but this does give me some more things to try once I have some more time.
